.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+Pull requests into this repository require the following checks to be completed by the submitter and reviewers.
+
+**Submitter:**
+- [ ] Administrative tasks are complete.
+  - This pull request describes why these changes were made.
+  - The JIRA ticket links to this PR.
+  - The JIRA ticket for this PR is: 
+- [ ] Tests are complete.
+  - Tests are included and test edge cases.
+  - Tests have been run locally and pass.
+- [ ] Code review has been performed.
+  - Code diff has been done and checked to ensure only expected code is being committed.
```

Comment:
On a previous project I was on, it was a requirement to make sure your feature branch has rebased off of master to make sure you have all the latest changes before merging. That project in general favored rebasing and rewriting history over merging though; what's your opinion on rebasing vs merging?